src/pages/journal/journal.actions.ts

```diff
@@ -37,11 +41,26 @@ export class ClearChangedSlot implements Action {
   constructor(public slotId: number) {}
 }
 
+export class SelectPreviousDay implements Action {
+  readonly type = SELECT_PREVIOUS_DAY;
+}
+
+export class SelectNextDay implements Action {
+  readonly type = SELECT_NEXT_DAY;
+}
+
+export class SetselectedDate implements Action {
+  readonly type = SET_SELECTED_DAY;
```

Comment:
One I missed - Can you fix the casing on this please :)

Reply:
Good spot!